src/server/src/main/java/io/cassandrareaper/jmx/ClusterFacade.java

```diff
@@ -772,8 +773,34 @@ public void clearSnapshot(String snapshotName, Node host) throws ReaperException
   private Map<List<String>, List<String>> getRangeToEndpointMapImpl(
       Cluster cluster,
       String keyspace) throws ReaperException {
-
-    return connect(cluster).getRangeToEndpointMap(keyspace);
+    JmxProxy jmxConnection = connect(cluster);
+    Map<List<String>, List<String>> endpointMap = jmxConnection.getRangeToEndpointMap(keyspace);
+    return endpointCleanup(endpointMap);
+  }
+
+  protected static Map<List<String>, List<String>> endpointCleanup(Map<List<String>, List<String>> endpointMap) {
```

Comment:
The name of the method is not enough here to understand what it's really doing and we don't have comments either.
Is this method converting Scylla endpoint lists to be compatible with Cassandra? Is it doing more?

Reply:
The method makes the Scylla endpoint list compatible with the Cassandra ones but gets called also while using cassandra (this is the reason while I didn't name it scyllaEndpointCleanup). Do you have any suggestion on how to make it more clear?

Reply:
Adding what you just wrote in an element comment on the method would do it :)
You could also rename the method to `maybeCleanupEndpointFromScylla()` as the method may or may not do something (I understand it's a noop for Cassandra).

Reply:
Looks fair, thanks!
Applied changes as you suggested.